src/components/DpDatepicker/DpDatepicker.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta } from '@storybook/addon-docs'
+import DpDatepicker from './'
+
+<Meta
+    title="Components/Datepicker"
+    component={DpDatepicker}
+/>
+
+# Datepicker
+
+The Datepicker component use `a11y-datepicker` component.
+
+## Default usage
+
+```html
+<dp-datepicker
+  id="datepicker_id" />
```

Comment:
To have a more realistic example, i'd suggest to add `v-model="planDate"` here.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/521/commits/1c48425c019ec7a0de619a4b3ccf39b2a36a0925